packages/documentation/src/pages/platform/unit-testing.mdx

```diff
@@ -1,18 +1,30 @@
 # Unit testing
 
 <div class="deprecation-message">
-    <h2>We're moving our docs!</h2>
-    <h3>Find <a href="https://depo-platform-documentation.scrollhelp.site/developer-docs/Unit-tests.1836187655.html">the latest version of this page</a> on the Platform website.</h3>
-    <h3>Still can't find what you're looking for? Reach out to <a href="https://dsva.slack.com/archives/CBU0KDSB1">#vfs-platform-support</a> on Slack.</h3>
+  <h2>We're moving our docs!</h2>
+  <h3>
+    Find{" "}
```

Comment:
@it-harrison Does this work in a markdown file? Wonder if the auto-formatter went haywire

Reply:
Ah okay, it's a `.mdx` file so that seems fine